lib/bike_brigade_web/live/campaign_signup_live/show.ex

```diff
@@ -172,7 +172,12 @@ defmodule BikeBrigadeWeb.CampaignSignupLive.Show do
 
   defp assign_campaign(socket, campaign) do
     {riders, tasks} = Delivery.campaign_riders_and_tasks(campaign)
-    tasks = Enum.sort_by(tasks, fn t -> t.dropoff_location.neighborhood.name end)
+    tasks = Enum.sort_by(tasks, fn t ->
```

Comment:
Why not use `Location.neighborhood` here which handles nils?

Reply:
Ooh, I forgot.